Formula/rust.rb

```diff
@@ -11,6 +11,11 @@ class Rust < Formula
       url "https://github.com/rust-lang/cargo.git", :tag => "0.14.0", :revision => "eca9e159b6b0d484788ac757cf23052eba75af55"
     end
 
+    resource "racer" do
+      url "https://github.com/phildawes/racer/archive/1.2.5.tar.gz"
```

Comment:
I'm a bit confused by the released here: https://github.com/phildawes/racer/releases

1.1.0 is marked as latest and 1.2.5 is just a tag. But in the just-a-tag category, there's also 2.0.1, 2.0.2, and 2.0.3. Do we have to wait for https://github.com/Homebrew/homebrew-core/pull/8138 to get fixed before using 2.0.x? And if that's the case, what's the rationale for 1.2.5 over 1.1.0?